test/msd_test.py

```diff
@@ -373,6 +375,56 @@ def test_mass_storage(workspace, parent_test):
     #        when a binary file is loaded, since there is no way to
     #        tell where the end of the file is.
 
+    if page_erase_supported:
+        # Test page erase, a.k.a. sector erase by generating iHex with discrete addresses,
+        # programing the device then comparing device memory against expected content.
+        test = MassStorageTester(board, test_info, "Sector Erase")
+        with MbedBoard.chooseBoard(board_id=board.get_unique_id(), init_board=False) as mbed_board:
+            memory_map = mbed_board.target.getMemoryMap()
+        flash_regions = [region for region in memory_map if region.type == 'flash']
+        number_flash_regions = len(flash_regions)
+
+        max_address = intel_hex.maxaddr()
+        # Create an object. We'll add the addresses of unused even blocks to it first, then unused odd blocks for each region
+        ih = intelhex.IntelHex()
+        # Add the content from test bin first
+        expected_bin_contents = bin_file_contents
+        for region_index in range(0, number_flash_regions, 1):
+            flash_start = flash_regions[region_index].start
+            flash_length = flash_regions[region_index].length
+            block_size = flash_regions[region_index].blocksize
+            number_of_blocks = int(flash_length/block_size)
```

Comment:
Use the `//` operator instead of the `int` constructor.

Reply:
Got it, will change. Thanks for letting me know.